node-js-peer/package.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
This update is not required I suppose

Reply:
@flcl42 , I think changes from other peers might be because I synced my folked repo.

Reply:
@dhuseby could you merge main into dotnet-peer-development please?